src/cascadia/TerminalApp/Command.idl

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "../ShortcutActionDispatch.idl";
+
+namespace TerminalApp
+{
+    [default_interface] runtimeclass Command : Windows.UI.Xaml.Data.INotifyPropertyChanged

```

Comment:
very generic name, consider further qualification, `TerminalCommand`

Reply:
This is not a namespace that is intended to be exposed to external consumers, and I've always held that WinRT's WinRTRepeatedNamespacesInClassNames and RepeatedNamespaceEnum::RepeatedNamespaceEnumValueOkay were a misfeature :smile:

Is this concern necessary for internal implementation details?